rr_openrover_driver/src/odom_control.cpp

```diff
@@ -272,30 +276,38 @@ int OdomControl::deadbandOffset(int motor_speed, int deadband_offset)
   }
 }
 
-double OdomControl::filter(double velocity, double dt)
+double OdomControl::filter(double velocity, double dt, int firmwareBuildNumber)
 {
   static double time = 0;
-  float change_in_velocity = 0;
 
   // Check for impossible acceleration, if it is impossible, ignore the measurement.
   float accel = (velocity - velocity_filtered_history_[0]) / dt;
   velocity_history_.insert(velocity_history_.begin(), velocity);
   velocity_history_.pop_back();
-
-  if (accel > MAX_ACCEL_CUTOFF_)
-  {
-    change_in_velocity = 0.5*dt*MAX_ACCEL_CUTOFF_;
-    velocity = velocity_filtered_history_[0] + change_in_velocity;
+  
+  if(firmwareBuildNumber == BUILD_NUMBER_WITH_GOOD_RPM_DATA){
+    std::cout << "switching correctly" << std::endl;
```

Comment:
did you mean to check this in? it looks like debug output

Reply:
I'll cleanup the std::out prints. There are a few others within the PR as well.